main/rpc/index.js

```diff
@@ -227,6 +229,20 @@ const rpc = {
   resolveAragonName (name, chainId, cb) {
     resolveName(name, chainId).then(result => cb(null, result)).catch(cb)
   },
+  resolveEnsName (name, cb) {
+    log.info('resolving name ', name)
+    const nebula = nebulaApi()
+    nebula.ens.resolve(name).then(result => {
+      const {addresses: {eth: ethAddress}} = result
+      if(!ethAddress){
```

Comment:
don't use errors for control flow. we already know how to handle this case so we can either just callback with undefined and let the calling code handle it callback with the error